src/js/helpers/searchIndexBuilder.js

```diff
@@ -0,0 +1,73 @@
+const searchIndexBuilder = function(pages) {
+
```

Comment:
The thinking behind creating this as a separate page was merely to make it easier to find/return to when modifying the search Index in the future.